components/page.js

```diff
@@ -45,12 +45,20 @@ export default class extends Component {
     let dataSet = new Data(this.props.data);
     let histogramArray = []; // JSON.parse(this.props.histogram);
     let timestamp = this.props.timestamp ? this.props.timestamp : 0;
-    let xmin = this.props.xmin;
-    let xmax = this.props.xmax;
+
+    const elos = dataSet.getRatingsArray(this.props.dataLabelOne);
+    const teamElos = dataSet.getRatingsArray(this.props.dataLabelTwo);
+
+    // Share xmin and xmas for all 3 graphs
+    const xmin = Math.min(...[...elos, ...teamElos].filter(Boolean)).clamp(0, Infinity);
+    const xmax = Math.max(...[...elos, ...teamElos].filter(Boolean)).clamp(0, Infinity);
+
+    const xbins = {start: 0, end: 3500,size: 10}; // each bin covers 10 elo points
```

Comment:
Page crashes if you attempt to lookup a user who managed to get their rating < 0 or >=3500 (probably a cheater/manipulator?). The histogram highlight can't find the right bucket to highlight and doesn't handle it well. 

Not a problem now, but I think I'll add an update in a separate commit to just not highlight anything instead of crashing, just in case things shift in the future.